  INFO root [2019-06-19 19:33:00,098] - 
  INFO root [2019-06-19 19:33:00,101] - -----------------------------------------------------------------------------------------


  INFO root [2019-06-19 19:33:00,101] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-06-19 19:33:00,101] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/HappyPoker_assets/;D:/kbengine/HappyPoker_assets/scripts/;D:/kbengine/HappyPoker_assets/res/
  INFO root [2019-06-19 19:33:00,101] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-06-19 19:33:00,112] - NetworkInterface::initialize(EXTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:20013 )
  INFO root [2019-06-19 19:33:00,112] - NetworkInterface::initialize(EXTERNAL-TCP): address 192.168.56.1:20013, SOMAXCONN=128.
  INFO root [2019-06-19 19:33:00,118] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.56.1 ( 192.168.56.1:49809 )
  INFO root [2019-06-19 19:33:00,118] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.56.1:49809, SOMAXCONN=128.
  INFO root [2019-06-19 19:33:00,118] - server-configs:
  INFO root [2019-06-19 19:33:00,118] - 	internalTcpAddr : 192.168.56.1:49809
  INFO root [2019-06-19 19:33:00,118] - 	externalTcpAddr : 192.168.56.1:20013
  INFO root [2019-06-19 19:33:00,118] - 	componentID : 9000
  WARN root [2019-06-19 19:33:00,322] - invalid UID(0) <= 0, please check UID for environment! automatically set to 3960.
  INFO root [2019-06-19 19:33:00,648] - Components::findLogger: found logger, addr:192.168.56.1:49806
  INFO root [2019-06-19 19:33:00,648] - Components::addComponent: reset orderLog, uid=3960!
  INFO root [2019-06-19 19:33:00,648] - Components::addComponent[logger], uid=3960, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-06-19 19:33:00,648] - ---- loginapp Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CE9FF53F4E85387EE4455C7191269324. Config: Debug X86. Built: 09:58:16 May  8 2019. AppID: 9000. UID: 3960. PID: 3652 ----
  INFO root [2019-06-19 19:33:00,648] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-06-19 19:33:00,648] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-06-19 19:33:00,648] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-06-19 19:33:00,851] - Script::install(): is successfully, Python=(3.7.3 (default, May  8 2019, 09:53:38) [MSC v.1916 32 bit (Intel)])!
  INFO root [2019-06-19 19:33:00,851] - Script::install(): KBExtra is successfully!
S_INFO root [2019-06-19 19:33:00,858] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2019-06-19 19:33:00,858] - TelnetServer server is running on port 31000
  INFO root [2019-06-19 19:33:01,221] - ---- loginapp is running ----
 DEBUG root [2019-06-19 19:33:01,221] - Components::process(): Request for the process of identity...
  WARN root [2019-06-19 19:33:07,448] - Forwarding logs to logger[192.168.56.1:49806/0/0/0]...
 ERROR root [2019-06-19 19:33:11,713] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=1016701816, lastReceivedTime=991790303, inactivityExceptionPeriod=134887500)
  INFO root [2019-06-19 19:33:11,713] - The following logs sent to logger failed:
  WARN root [2019-06-19 19:33:11,713] - ==> [2019-06-19 19:33:11 721] TCPPacketReceiver::processPendingEvents(192.168.56.1:49806): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-06-19 19:33:11,713] - ==> [2019-06-19 19:33:11 722] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=1016701816, lastReceivedTime=991790303, inactivityExceptionPeriod=134887500)
